src/data_structure/field2D.java

```diff
@@ -46,6 +58,41 @@ public Type getType() {
         return type;
     }
 
+    public void setColumn(fieldInfo column) { // TODO: (kam193) Maybe we should raise exception, when operation is not permitted
+        this.column = column;
+        if (this.type == Type.BLANK)
+            this.type = Type.INFOCOLUMN;
+        else if (this.type == Type.INFOROW)
+            this.type = Type.INFOCOLUMNANDROW;
+    }
+
+    public void setRow(fieldInfo row) {
+        this.row = row;
+        if (this.type == Type.BLANK)
+            this.type = Type.INFOROW;
+        else if (this.type == Type.INFOCOLUMN)
+            this.type = Type.INFOCOLUMNANDROW;
+    }
+
+    @Override
+    public String toString(){
+        if (this.type == Type.BLANK){
+            return "#";
```

Comment:
Create final char or string instead of using magic strings.

Reply:
I think in `toString()` methods we can use magic strings